docs/features/mention.md

```diff
@@ -0,0 +1,56 @@
+---
+category: features
+---
+
+{@snippet features/build-mention-source}
+
+# Mention
+
+The {@link module:mention/mention~Mention} feature brings support for smart completion based on user input. When user types a pre-configured marker, such as `@` or `#`, they get an autocomplete suggestions in a balloon panel displayed next to a caret. The selected suggestion is inserted to the content.
+
+## Demo
+
+{@snippet features/mention}
```

Comment:
We need a description what this demo is about, what to do, what to type, etc..

Reply:
Sure. Right now it's a draft only. I'm working on it as a follow-up.